### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory:

### ISSUE THREAD

**Issue by OpenCode** [acsone/setuptools-odoo#116]
setuptools-odoo version: 3.2.0
SO: Ubuntu 22.04.3 LTS

If I use

`setuptools-odoo-make-default -c -d .` 

I obtain

```
Traceback (most recent call last):
  File "/usr/local/bin/setuptools-odoo-make-default", line 8, in <module>
    sys.exit(main())
  File "/home/xxxxxxxxxx/.local/lib/python3.10/site-packages/setuptools_odoo/make_default_setup.py", line 416, in main
    clean_setup_addons_dir(args.addons_dir, args.odoo_version_override)
  File "/home/xxxxxxxxxx/.local/lib/python3.10/site-packages/setuptools_odoo/make_default_setup.py", line 326, in clean_setup_addons_dir
    os.unlink(p)
FileNotFoundError: [Errno 2] No such file or directory: './setup/MY_MODULE/odoo_addons'
```

If I delete manually the directory "setup" I have the same error.

With version 3.1.8 everything is fine

**Comment by sbidoul** [comment]
Hm, this is likely a regression from #114

**Comment by sbidoul** [comment]
Fix in #117
